docs/pipelines/process/repository-resource.md

```diff
@@ -42,7 +42,7 @@ You must be a member of the [**Project Administrators** group](../../organizatio
 Your repository has a resource check.
 ## Add pipeline permissions to a repository resource
 
-You can also set a repository to only be used on specific pipelines. Restricting a repository to specific pipelines prevents an unauthorized pipeline in your project from using your repository.
+You can also set a repository to only be used on specific pipelines ?? is this true for classic ??. Restricting a repository to specific pipelines prevents an unauthorized pipeline in your project from using your repository.
```

Comment:
@vijayma Can you look at this specifically?

Reply:
This only works for YAML pipelines.

- Generating a restrictive token is based on what the user declares in the resources section of the YAML pipeline. Such a section can only be defined in YAML pipelines. There is no provision to define this in classic pipelines. We did not implement this behavior for classic pipelines since there is just no mechanism we could give them for generating a broader token.
- Pipeline permissions also works only for YAML pipelines.

So, here also, we can say that this only works for YAML.